### PR TITLE
Prevent unhandled exception when running via schedule

### LINKED ISSUE #68: Error when triggering via a Schedule event
When using this Action in a workflow that is triggered by a Scheduled event (ie cron), there is an unhandled exception if `PULL_REQUEST_FROM_BRANCH` is not set.

**Error Message**
```
PULL_REQUEST_FROM_BRANCH is not set, checking branch in payload.
Found ${GITHUB_EVENT_PATH} at /github/workflow/event.json
Traceback (most recent call last):
  File "/pull-request.py", line 454, in <module>
    main()
  File "/pull-request.py", line 398, in main
    from_branch = from_branch.replace("refs/heads/", "").strip("/")
AttributeError: 'NoneType' object has no attribute 'replace'
```

**How to reproduce**

```
name: "Create PR"

on:
  push:
    branches:
      - develop
  schedule:
    - cron: '* * * * *'

jobs:
  provisioning:
    name: "Install Infrastructure"
    runs-on: ubuntu-latest
    steps:
      - name: "Setup - Checkout Code"
        id: checkout-code
        uses: actions/checkout@v2.3.4

      - name: Create PR for deploying to Prod
        uses: vsoch/pull-request-action@1.0.15
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          BRANCH_PREFIX: "develop"
          PULL_REQUEST_BRANCH: "master"
```

I will be submitting a PR to resolve this issue momentarily.

### PR DESCRIPTION
**Pull request by mcbanderson**
Resolves #68

### CODE REVIEW

File: pull-request.py [hunk 1]
```diff
@@ -395,7 +395,10 @@ def main():
         print("PULL_REQUEST_FROM_BRANCH is not set, checking branch in payload.")
         with open(check_events_json(), "r") as fd:
             from_branch = json.loads(fd.read()).get("ref")
```
Comment (vsoch) [inline pull-request.py:397]: Thank you for catching this! A question about your choice of fix - why not just do:

```suggestion
            from_branch = json.loads(fd.read()).get("ref", "")
```
And avoid the extra try / except and exception? The result of the above is an empty string, which won't trigger the error, and will cleanly exit in the next block telling the user to set the envar.

Comment (mcbanderson) [reply pull-request.py:397]: Good point, that is cleaner. Updating now

**Review by vsoch — APPROVED** [review]
Perfect! Let me get this merged and released as a version so you can use it asap. Thank you again!

**Comment by mcbanderson** [comment]
Happy to help, thanks for the quick response and merge! There's no need to cut a new release just for this (unless that's your usual workflow). Either way I needed to update my Workflow to utilize `PULL_REQUEST_FROM_BRANCH`, which I've already done

**Comment by vsoch** [comment]
I'm of the mindset to release often, so it's my preference to do a quick release. You should be good to go without it, but for provenance in this discussion, I'll link it here: https://github.com/vsoch/pull-request-action/releases/tag/1.0.16. Happy coding! <3

**Pull request merged**